pkg/fakerp/config.go

```diff
@@ -88,19 +91,23 @@ func GetPluginConfig() (*api.PluginConfig, error) {
 	}
 
 	// populate geneva artifacts
-	artifactDir := "secrets/"
-	logCert, err := readCert(artifactDir + "logging-int.cert")
+	artifactDir, err := config.FindDirectory(LoggingSecretsDirectory)
 	if err != nil {
 		return nil, err
 	}
-	logKey, err := readKey(artifactDir + "logging-int.key")
+	logCert, err := readCert(path.Join(artifactDir, "logging-int.cert"))
 	if err != nil {
 		return nil, err
 	}
-	pullSecret, err := readFile(artifactDir + ".dockerconfigjson")
+	logKey, err := readKey(path.Join(artifactDir, "logging-int.key"))
 	if err != nil {
 		return nil, err
 	}
+	pullSecret, err := readFile(path.Join(artifactDir, ".dockerconfigjson"))
```

Comment:
path.Join is not compatible with Windows, use filepath.Join

Reply:
good to know